src/runtimes/node/index.ts

```diff
@@ -67,6 +59,17 @@ const zipFunction: ZipFunction = async function ({
     return { config, path: destPath }
   }
 
+  const inSourceConfig = await findISCDeclarationsInPath(mainFile, name, featureFlags)
```

Comment:
I'd like to call out this change, since we're now parsing ISC before bundling. This means we'll start parsing ESM, CJS and TS rather than the bundled/transpiled CJS. I don't anticipate any problems with this, but it's not something I can easily wrap with a feature flag, so something to keep an eye on.